README.md

```diff
@@ -1,70 +1,85 @@
-# Getting Started with Create React App
+# Space-Travelers-Hub
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This is a web application for a company that provides commercial and scientific space travel services. The application will allow users to book rockets and join selected space missions.
 
-## Available Scripts
+## Built With
 
-In the project directory, you can run:
+- React 
+- CSS
+- React-bootstrap 
+- Axios
+- JSX
 
-### `npm start`
+## Live Demo
+### Comming soon
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Screenshot 
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+### Comming Soon
+## Getting Started
 
-### `npm test`
+### Prerequisites
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+- You should be having [Node.js](https://nodejs.org/en/) installed on your computer
+### Setup
 
-### `npm run build`
+To get a local copy up and running follow these simple example steps.
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+1. Navigate to a folder where you want the cloned file to appear
+   
+2. Clone this repo in your terminal or git bash using the command
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+  https://github.com/melashu/Space-Travelers-Hub.git
 
```

Comment:
- [ ] I would recommend including the step on how to test the application. Hint : `npm test`

Reply:
Thank you @Deepakdanger for your feedback. But testing is not part of the requirement.